ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-devpreview.yaml

```diff
@@ -1130,16 +1057,16 @@ tests:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       COMPUTE_NODE_TYPE: Standard_D4ps_v5
       E2E_RUN_TAGS: '@arm64 and @azure-ipi and @disconnected and @network-ovnkubernetes
-        and not @fips and @destructive'
-      FEATURE_SET: TechPreviewNoUpgrade
+        and not @fips and @destructive and @long-duration'
       OCP_ARCH: arm64
       TAG_VERSION: '@4.14'
-      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      TEST_SCENARIOS: Cluster_Infrastructure
+      TEST_TIMEOUT: "90"
     test:
-    - chain: openshift-e2e-test-qe-destructive
-    workflow: cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate
-- as: azure-ipi-disconnected-f14
-  cron: 22 20 9,24 * *
+    - chain: openshift-e2e-test-qe-long-duration
+    workflow: cucushift-installer-rehearse-azure-ipi-disconnected
+- as: azure-ipi-disconnected-fullyprivate-f14
```

Comment:
Pls do not change the order if it is not related to this PR's change.

Reply:
I just tried to maintain the sort order, between them ..
`azure-ipi-disconnected-f28-long-duration-cloud` before` azure-ipi-disconnected-fullyprivate-f14`
thought '2' is before 'u' , can you suggest ?